pkg/virt-operator/util/client.go

```diff
@@ -74,7 +126,6 @@ func UpdateCondition(kv *virtv1.KubeVirt, conditionType virtv1.KubeVirtCondition
 	}
 
 	kv.Status.Conditions = conditions
-
 }
 
 func getCondition(kv *virtv1.KubeVirt, conditionType virtv1.KubeVirtConditionType) (*virtv1.KubeVirtCondition, bool) {
```

Comment:
Is it possible for this to be made public so the HCO can get the conditions?

Reply:
I leave it private. It does something you don't need (creating a condition if it does not exist), and I want to be able to change it if needed, without breaking HCO.
I think copying the few lines you need to HCO is good enough :slightly_smiling_face: